manim/animation/animation.py

```diff
@@ -60,9 +57,9 @@ def __init__(
                 )
             )
 
-    def _typecheck_input(self, mobject: Mobject) -> None:
+    def _typecheck_input(self, mobject: Union[Mobject, None]) -> None:
```

Comment:
Optional[Mobject] ?

Reply:
Had considered Optional, but since "None" is truly an exception, and "Mobject" is the norm ... 
I think Optional is optional :)

Reply:
sounds good"